### PR TITLE
Mobile ui and refactored filters #1438

### PR DESCRIPTION
**Pull request by michaelDpierce**

### CODE REVIEW

File: app/views/certifications/index.html.erb [hunk 1]
```diff
@@ -86,10 +85,7 @@
         </tbody>
       </table>
     </div>
-    <div ng-hide='(users
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length'>
+    <div ng-hide='filteredUsers.length'>
```
Comment (duffcodester) [inline app/views/certifications/index.html.erb]: nice!

File: app/views/help_desks/closed_index.html.erb [hunk 2]
```diff
@@ -36,11 +33,11 @@
           </tr>
         </thead>
         <tbody>
-          <tr ng-repeat='item in closedHelpDesks
+          <tr ng-repeat='item in filtedClosedHelpDesks = (closedHelpDesks
```
Comment (duffcodester) [inline app/views/help_desks/closed_index.html.erb]: rename item to helpDesk or issue or something that makes more sense

File: app/views/help_desks/index.html.erb [hunk 3]
```diff
@@ -42,11 +39,11 @@
           </tr>
         </thead>
         <tbody>
-          <tr ng-repeat='item in helpDesks
+          <tr ng-repeat='item in filtedHelpDesks = (helpDesks
```
Comment (duffcodester) [inline app/views/help_desks/index.html.erb]: same here

File: app/views/help_desks/closed_index.html.erb (hunk 2 above)
Comment (michaelDpierce) [inline app/views/help_desks/closed_index.html.erb]: Done

File: app/views/help_desks/index.html.erb (hunk 3 above)
Comment (michaelDpierce) [inline app/views/help_desks/index.html.erb]: Done

File: app/views/help_desks/closed_index.html.erb [hunk 4]
```diff
@@ -19,15 +19,12 @@
       </div>
     </div>
     <br>
-    <div ng-show='(closedHelpDesks
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length' class='shadow-effect'>
+    <div ng-show='filtedItems.length' class='shadow-effect'>
```
Comment (duffcodester) [inline app/views/help_desks/closed_index.html.erb]: say filteredHelpDesks

File: app/views/help_desks/closed_index.html.erb [hunk 5]
```diff
@@ -36,11 +33,11 @@
           </tr>
         </thead>
         <tbody>
-          <tr ng-repeat='item in closedHelpDesks
+          <tr ng-repeat='item in filtedItems = (closedHelpDesks
```
Comment (duffcodester) [inline app/views/help_desks/closed_index.html.erb]: match here: filteredHelpDesks

File: app/views/help_desks/closed_index.html.erb [hunk 6]
```diff
@@ -68,16 +65,10 @@
         </tbody>
       </table>
     </div>
-    <div ng-show='(closedHelpDesks
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length > 10'>
+    <div ng-show='filtedItems.length > 10'>
```
Comment (duffcodester) [inline app/views/help_desks/closed_index.html.erb]: match here

File: app/views/help_desks/closed_index.html.erb [hunk 7]
```diff
@@ -68,16 +65,10 @@
...
       <ng-include src="'load-more.html'"></ng-include>
     </div>
-    <div ng-hide='(closedHelpDesks
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length'>
+    <div ng-hide='filtedItems.length'>
```
Comment (duffcodester) [inline app/views/help_desks/closed_index.html.erb]: and here

File: app/views/help_desks/index.html.erb [hunk 8]
```diff
@@ -26,10 +26,7 @@
       <div><i class='fa fa-star star'></i> <span class='grey-text'> New</span></div>
     </div>
     <br>
-    <div ng-show='(helpDesks
-                  | orderBy:order.value:order.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length' class='shadow-effect'>
+    <div ng-show='filtedItems.length' class='shadow-effect'>
```
Comment (duffcodester) [inline app/views/help_desks/index.html.erb]: filteredHelpDesks

File: app/views/help_desks/index.html.erb [hunk 9]
```diff
@@ -42,11 +39,11 @@
           </tr>
         </thead>
         <tbody>
-          <tr ng-repeat='item in helpDesks
+          <tr ng-repeat='item in filtedItems = (helpDesks
```
Comment (duffcodester) [inline app/views/help_desks/index.html.erb]: filteredHelpDesks

File: app/views/help_desks/index.html.erb [hunk 10]
```diff
@@ -75,16 +72,10 @@
         </tbody>
       </table>
     </div>
-    <div ng-show='(helpDesks
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length > 10'>
+    <div ng-show='filtedItems.length > 10'>
```
Comment (duffcodester) [inline app/views/help_desks/index.html.erb]: filteredHelpDesks

File: app/views/help_desks/index.html.erb [hunk 11]
```diff
@@ -75,16 +72,10 @@
...
       <ng-include src="'load-more.html'"></ng-include>
     </div>
-    <div ng-hide='(helpDesks
-                  | orderBy:predicate.value:predicate.reverse
-                  | filter:{location_id: predicate.locationFilter || undefined}
-                  | filter:predicate.search).length'>
+    <div ng-hide='filtedItems.length'>
```
Comment (duffcodester) [inline app/views/help_desks/index.html.erb]: filteredHelpDesks

Comment (michaelDpierce) [inline app/views/help_desks/index.html.erb]: Done

File: app/views/help_desks/index.html.erb (hunk 10 above)
Comment (michaelDpierce) [inline app/views/help_desks/index.html.erb]: Done

File: app/views/help_desks/index.html.erb (hunk 9 above)
Comment (michaelDpierce) [inline app/views/help_desks/index.html.erb]: Done

File: app/views/help_desks/index.html.erb (hunk 8 above)
Comment (michaelDpierce) [inline app/views/help_desks/index.html.erb]: Done

File: app/views/help_desks/closed_index.html.erb (hunk 7 above)
Comment (michaelDpierce) [inline app/views/help_desks/closed_index.html.erb]: Done

File: app/views/help_desks/closed_index.html.erb (hunk 6 above)
Comment (michaelDpierce) [inline app/views/help_desks/closed_index.html.erb]: Done

File: app/views/help_desks/closed_index.html.erb (hunk 5 above)
Comment (michaelDpierce) [inline app/views/help_desks/closed_index.html.erb]: Done

File: app/views/help_desks/closed_index.html.erb (hunk 4 above)
Comment (michaelDpierce) [inline app/views/help_desks/closed_index.html.erb]: Done

**Comment by michaelDpierce** [comment]
Tested on staging

**Pull request merged**